pkg/mkimage-raw-efi/install

```diff
@@ -168,12 +168,20 @@ zfs_umount() {
   umount /root/dev ||:
 }
 
+
+cmdline="/proc/cmdline"
 logmsg "EVE-OS installation started"
+if grep -q interactive /proc/cmdline; then
+  logmsg "Interactive mode"
+  bash ./parse-interactive-installer-conf
+  cmdline=/tmp/newcmdline
+fi
+
 # do this just in case
 modprobe usbhid && modprobe usbkbd
 # clean partition tables on disks defined to nuke
-if grep -q eve_nuke_disks /proc/cmdline; then
-  NUKE_DISKS=$(</proc/cmdline tr ' ' '\012' | sed -ne '/^eve_nuke_disks=/s#^.*=##p')
+if grep -q eve_nuke_disks "$cmdline"; then
+  NUKE_DISKS=$(<"$cmdline" tr ' ' '\012' | sed -ne '/^eve_nuke_disks=/s#^.*=##p')
```

Comment:
The input redirection from $cmdline might not make sense - echo "$cmdline" might be what you need.